services/storage/requirements/_test.in

```diff
@@ -5,6 +5,8 @@
 --constraint ../../../requirements/constraints.txt
 --constraint _base.txt
 
+# to test backwards compatibility against deprecated client-sdk (used still in old versions of simcore-sdk)
+simcore-service-storage-sdk @ git+https://github.com/ITISFoundation/osparc-simcore.git@cfdf4f86d844ebb362f4f39e9c6571d561b72897#subdirectory=services/storage/client-sdk/python

```

Comment:
I see you install the SDK that the legacy services use. This way you can test with their interface agains the new API. Great.